recipes/cern-root/all/conanfile.py

```diff
@@ -0,0 +1,316 @@
+                    "tmva-pymva": "OFF",  # FIXME: switch on if Python CCI package available
+                    "xrootd": "OFF",  # FIXME: switch on if xrootd CCI package available
+                    "pyroot": self._pyrootopt,
+                    # Tell CMake where to look for Conan provided depedencies
+                    "CMAKE_LIBRARY_PATH": cmakelibpath.replace("\\", "/"),
+                    "CMAKE_INCLUDE_PATH": cmakeincludepath.replace("\\", "/"),
+                    # Configure install directories
+                    # Conan CCI hooks restrict the allowed install directory
+                    # names but ROOT is very picky about where build/runtime
+                    # resources get installed.
+                    # Set install prefix to work around these limitations
+                    # Following: https://github.com/conan-io/conan/issues/3695
+                    "CMAKE_INSTALL_PREFIX": os.sep.join((self.package_folder, "res")),
+                    # Fix some Conan-ROOT CMake variable naming differences
+                    "PNG_PNG_INCLUDE_DIR": ";".join(
+                        self.deps_cpp_info["libpng"].include_paths
+                    ).replace("\\", "/"),
+                    "LIBLZMA_INCLUDE_DIR": ";".join(
+                        self.deps_cpp_info["xz_utils"].include_paths
+                    ).replace("\\", "/"),
+                },
+            )
+        return self._cmake
+
+    def _move_findcmake_conan_to_root_dir(self):
+        for f in ["opengl_system", "GLEW", "glu", "TBB", "LibXml2", "ZLIB", "SQLite3"]:
+            shutil.copy(
+                "Find{}.cmake".format(f),
+                os.sep.join((self.source_folder, self._rootsrcdir, "cmake", "modules")),
+            )
+
+    @property
+    def _cmake_cxx_standard(self):
+        compileropt = self.settings.compiler.get_safe("cppstd")
+        if compileropt:
+            return str(compileropt)
+        else:
+            return "11"
+
+    @property
+    def _pyrootopt(self):
+        if self.options.python == PythonOption.OFF:
+            return "OFF"
+        else:
+            return "ON"
+
+    def build(self):
+        self._fix_source_permissions()
+        self._patch_source_cmake()
+        self._configured_cmake.build()
+
+    def package(self):
+        self._configured_cmake.install()
+        self.copy("LICENSE.txt", dst="licenses")
+        for dir in ["include", "lib", "bin"]:
+            os.symlink(
+                os.sep.join((self.package_folder, "res", dir)),
+                os.sep.join((self.package_folder, dir)),
+            )
+        # Fix for CMAKE-MODULES-CONFIG-FILES (KB-H016)
+        tools.remove_files_by_mask(self.package_folder, "*Config*.cmake")
+        # Fix for CMAKE FILE NOT IN BUILD FOLDERS (KB-H019)
+        os.remove(
+            os.sep.join((self.package_folder, "res", "tutorials", "CTestCustom.cmake"))
+        )
+
+    def package_info(self):
```

Comment:
Perhaps add a FIXME that ROOT generates multiple cmake config files?

Reply:
Apologies, I don't understand this suggestion.

The fact that ROOT outputs contains some `*Config.cmake` files is a feature (from ROOT's point of view).
Conan's linter doesn't want them in the output package to prevent a user accidentally using ROOT's `*Config.cmake` when the user should. 

So there is nothing to fix here?

Reply:
You may need to add an exception against the hooks repo for move them to a folder and have `cpp_info.builddirs` set correctly

https://docs.conan.io/en/latest/reference/conanfile/attributes.html#cpp-info

Reply:
Yes, removing those files is the right think to do according to CCI policy. But what @madebr says is that a FIXME could be added in `package_info()` to properly mimic multiple module/config files in `cmake_find_package[_multi]` generators (currently it's not possible 1 recipe => 1 config file for cmake_find_package_multi).

Reply:
Yes, removing those files is the right think to do according to CCI policy. But what @madebr says is that a FIXME could be added in `package_info()` to properly mimic multiple module/config files in `cmake_find_package[_multi]` generators (currently it's not possible 1 recipe => 1 config file for cmake_find_package_multi).

Reply:
Thanks. I think that I understand now. I will add an appropriate FIXME comment in my commit.

Reply:
This was added in 7ad4381b.